examples/advanced_circuits_algorithms/Grover/Grover.ipynb

```diff
@@ -410,10 +410,10 @@
    "outputs": [],
```

Comment:
<p>nit: We might have to re-run these cells since those are displaying output from SV1 device instead of LocalSimulator.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>